### PR TITLE
Sample-sheet assay-type will now be verified.

### PR DESCRIPTION
**Pull request by charles-cowart**
Legal values are currently ('TruSeq HT', 'Metagenomic').
Popular alternative 'Metagenomics' will now return an error to the user.

### CODE REVIEW

**Comment by charles-cowart** [comment]
Hi @wasade! Since @antgonza is away, would you mind reviewing this pull-request please? If and when it's approved I will test it on qiita-rc and then introduce it to production as a hot-fix. We are getting a number of sample-sheet submissions with a value that is no longer valid and causes a cryptic error message to be returned. Thanks!

**Comment by charles-cowart** [comment]
Although the sanity-check in QCJob should remain, I realized it would be better to detect and report the error when sample-sheet validation is performed.

File: sequence_processing_pipeline/QCJob.py [hunk 1]
```diff
@@ -202,6 +202,11 @@ def _process_sample_sheet(self):
 
         header = valid_sheet.Header
         chemistry = header['chemistry']
+
+        if header['Assay'] not in ['TruSeq HT', 'Metagenomic']:
```
Comment (wasade) [inline sequence_processing_pipeline/QCJob.py:206]: Given the list of valid assays is used here, and in Pipeline.py:218, can it be centralized as a variable and imported?

**Review by wasade — CHANGES_REQUESTED** [review]
Hi @charles-cowart, just a minor request to centralize a variable

File: sequence_processing_pipeline/QCJob.py (hunk 1 above)
Comment (charles-cowart) [reply sequence_processing_pipeline/QCJob.py]: great idea thanks!

**Comment by charles-cowart** [comment]
@wasade Thanks! Ready for review!

**Comment by wasade** [comment]
Thanks!!

**Pull request merged**